contracts/TokenHandler.sol

```diff
@@ -132,12 +132,16 @@ contract TokenHandler is ITokenHandler, ITokenManagerType, ReentrancyGuard, Crea
      * @param tokenManager The address of the token manager.
      */
     // slither-disable-next-line locked-ether
-    function postTokenManagerDeploy(uint256 tokenManagerType, address tokenManager) external payable {
-        // For lock/unlock token managers, the ITS contract needs an approval from the token manager to transfer tokens on its behalf
-        if (tokenManagerType == uint256(TokenManagerType.LOCK_UNLOCK) || tokenManagerType == uint256(TokenManagerType.LOCK_UNLOCK_FEE)) {
-            ITokenManager(tokenManager).approveService();
-        } else if (tokenManagerType == uint256(TokenManagerType.NATIVE_INTERCHAIN_TOKEN)) {
-            IMinter(ITokenManager(tokenManager).tokenAddress()).transferMintership(tokenManager);
+    function postTokenManagerDeploy(uint256 tokenManagerType, ITokenManager tokenManager) external payable {
+        // For native interhcain tokens we transfer mintership to the token manager.
+        // This is done here because InterchainToken bytecode needs to be fixed.
```

Comment:
```suggestion
        // This is done here because the InterchainToken bytecode is preferred to be fixed to avoid having multiple versions of the Token code used in production.
```